src/org/usfirst/frc/team166/robot/RobotMap.java

```diff
@@ -48,10 +48,10 @@
 
 		public static final int IntakeVictor = 9;
 		public static final int IntakeVictor2 = 10;
-
+		public static final int RollerVictor = 11;
 	}
 
-	// Encoder (Digital Input) Channels
+	// Encoder (Digital or Analog Input) Channels
```

Comment:
No. The point of these classes is that there's a separate section for analog inputs.